README.md

```diff
@@ -462,6 +462,11 @@ After you build and run the initial executable you will have to create another e
 
 ### Building and Running Demos
 
+Before building the demos, ensure you have installed the [prerequisite software](#prerequisites). On Ubuntu 18.04 and 20.04, `gcc`, `cmake`, and OpenSSL can be installed with:
+```sh
+sudo apt install build-essentials cmake libssl-dev
```

Comment:
I could be wrong but I believe that it should just be `build-essential` or do both work?
```suggestion
sudo apt install build-essential cmake libssl-dev
```

Reply:
It should be `build-essential`. One with s is not found in apt.

Reply:
Thanks